orchestration/dagster_orchestration/hca_orchestration/resources/slack.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+from dagster import configured, resource, String
+from dagster_slack import slack_resource
+
+
+class ConsoleSlackClient:
```

Comment:
👍 I like this pattern

Reply:
wondering if it'd be less readable to just directly decorate this class, i.e.
```python
@resource
class ConsoleSlackClient: ...
```

Reply:
🤷 up to you; I am 50/50 and would tie break toward consistency (@resource decorator on a wrapping function) in this case.

Reply:
shoutout to whoever you just tagged, i'm sure this happens to them quite a lot